WoWPro/WoWPro_Mapping.lua

```diff
@@ -573,8 +579,8 @@ function WoWPro:MapPoint(row)
 			-- autoarrival == 0 is a no-op
 		end
 		TomTom.db.profile.persistence.cleardistance = OldCleardistance
-	elseif TomTom then
-		WoWPro:print("WoWPro:MapPoint2(%s@%s/%s)",coords,tostring(zone),tostring(zm))
+	elseif Nx then
+		WoWPro:print("Carbonite WoWPro:MapPoint2(%s@%s/%s)",coords,tostring(zone),tostring(zm))
```

Comment:
I think that a better approach might be to forsake the TomTom compatibility layer and just call the Nx native interface.   Carbonite seems to not be emulating the new TomTom interface, and emulating the old interfaces is rather horrid, having tried to write the code myself for TomTom.  So I'd rather just go the Nx functions and not have another layer of confusion.

Reply:
`Nx:TTSetTarget (mid, zx, zy, opts["title"], opts["callbacks"])`

Reply:
Not forgotten.  Hope to merge my version in the next few weeks.
I apologize for the delay.

Reply:
Closing pull request.   Functionality implemented in https://github.com/Ludovicus-Maior/WoW-Pro-Guides/commit/d8717fe264ac3e003999b7029a69fd4a4f953cda#diff-267bc39b782785b0267ae52b40a4220b